ui/src/app/dashboard/views/DashboardConfigurationForm/DashboardConfigurationSubnetForm/DashboardConfigurationSubnetForm.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect } from "react";
+
+import { Spinner, Strip } from "@canonical/react-components";
+import { useDispatch, useSelector } from "react-redux";
+
+import FormikField from "app/base/components/FormikField";
+import FormikForm from "app/base/components/FormikForm";
+import LegacyLink from "app/base/components/LegacyLink";
+import baseURLs from "app/base/urls";
+import { actions as fabricActions } from "app/store/fabric";
+import fabricSelectors from "app/store/fabric/selectors";
+import { actions as subnetActions } from "app/store/subnet";
+import subnetSelectors from "app/store/subnet/selectors";
+import type { Subnet } from "app/store/subnet/types";
+
+type SubnetDiscoveryValues = {
+  [x: number]: Subnet["active_discovery"];
+};
+
+const DashboardConfigurationSubnetForm = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const subnets = useSelector(subnetSelectors.all);
+  const fabrics = useSelector(fabricSelectors.all);
+  const subnetsLoaded = useSelector(subnetSelectors.loaded);
+  const fabricsLoaded = useSelector(subnetSelectors.loaded);
+  const saved = useSelector(subnetSelectors.saved);
+  const saving = useSelector(subnetSelectors.saving);
+
+  useEffect(() => {
+    dispatch(subnetActions.fetch());
+    dispatch(fabricActions.fetch());
+  }, [dispatch]);
+
+  const loaded = subnetsLoaded && fabricsLoaded;
```

Comment:
@petesfrench Since this form requires both subnets and fabrics we should check that they're both loaded before rendering. Otherwise you might get the case where subnets have loaded but not fabrics, and the component doesn't know how to render the fabric link.